sample_files/.travis.yml

```diff
@@ -11,9 +11,21 @@ addons:
 
 env:
   global:
-  - VERSION="8.0" TESTS="0" LINT_CHECK="0"
+  - VERSION="8.0" TESTS="0" LINT_CHECK="0" TRANSIFEX="0"
+  - TRANSIFEX_USER='transbot@odoo-community.org'
+  # This line contains the encrypted transifex password
+  # To encrypt transifex password, use
+  # $ travis encrypt TRANSIFEX_PASSWORD=your-password -r owner/project
```

Comment:
I don't have the transbot password on hand, but I will provide you with the encrypted password for l10n-spain tomorrow, when I am at the office.